proc_Hahn_echoph.py

```diff
@@ -1,100 +1,72 @@
-from pyspecdata import *
-from scipy.optimize import leastsq, minimize, basinhopping
-from pyspecProcScripts import *
-from pyspecProcScripts.load_data import lookup_table
+import pyspecdata as psd
+import pyspecProcScripts as prscr
 from sympy import symbols
 
-fl = fl_mod()
 t2 = symbols("t2")
-logger = init_logging("info")
-
-
-def select_pathway(s, pathway):
-    retval = s
-    for k, v in pathway.items():
-        retval = retval[k, v]
-    return retval
+logger = psd.init_logging("info")
 
 
 signal_pathway = {"ph1": 1, "ph2": 0}
-for searchstr, exp_type, nodename, postproc, label_str, slice_f in [
-    (
-        "200302_alex_probe_water",
-        "test_equip",
-        "signal",
-        "spincore_Hahn_echoph_v1",
-        "microwaves off",
-        (-2.5e3, 2.5e3),
-    ),
-]:
-    # {{{loads raw data and plots
-    s = find_file(
-        searchstr,
-        exp_type=exp_type,
-        expno=nodename,
-        postproc=postproc,
-        lookup=lookup_table,
-        fl=fl,
-    )
-    s.mean("nScans")
-    # }}}
-    # {{{rough centering of sliced data
-    s = s["t2":slice_f]
-    s.ift("t2")
-    rough_center = (
-        abs(s).convolve("t2", 0.01).mean_all_but("t2").argmax("t2").item()
-    )
-    s.setaxis(t2 - rough_center)
```

Comment:
a lot of this is now done by fid_from_echo so I updated accordingly and made sure this runs

Reply:
1) we moved the data to the old folder so I just redirected it 2) I comment that the load data applies a squeeze which removes the nScans dimension and so the averaging throws an error 3) updated fid_from_echo replaces everything